src/CMakeLists.txt

```diff
@@ -240,6 +241,7 @@ if( MIOPEN_BACKEND MATCHES "OpenCL" OR MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN
     file(GLOB_RECURSE COMPOSABLE_KERNEL_INCLUDE "kernels/composable_kernel/include/*/*.hpp")
     file(GLOB_RECURSE COMPOSABLE_KERNEL_SOURCE "kernels/composable_kernel/src/*/*.cpp")
     file(GLOB_RECURSE COMPOSABLE_KERNEL_DYNAMIC_SOURCE "kernels/dynamic_igemm/*.s")
+    file(GLOB_RECURSE COMPOSABLE_KERNEL_DYNAMIC_AUX_SOURCE "kernels/dynamic_igemm/*.cpp")
```

Comment:
Perhaps, we should have named it as DYNAMIC_ASM_SOURCE  
and DYNAMIC_CPP_SOURCE

Reply:
Yes, renamed .